cg/store/api/add.py

```diff
@@ -149,6 +149,7 @@ def add_case(
         name: str,
         panels: List[str],
         priority: str = "standard",
+        **kwargs,
```

Comment:
Are you sure? Should be really hide what we put into the database

Reply:
maybe not, fixed